### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pandyaved98**
Added name for the Hindi Language Translation.

### CODE REVIEW

**Review by lewtun — APPROVED** [review]
Thank you @pandyaved98 ! I'll merge this once I've merged and deployed your PR with the translation (so that the URL works)

**Comment by pandyaved98** [comment]
It's still routing me to the English version of the course and while changing en to hi in the URL it gives 500 - Internal error.

**Review by lewtun — APPROVED** [review]

**Pull request merged**
